tests/integration/src/test/java/org/mule/test/core/context/notification/ConnectorMessageErrorNotificationTestCase.java

```diff
@@ -24,32 +21,32 @@
 public class ConnectorMessageErrorNotificationTestCase extends AbstractNotificationTestCase {
 
   private static final String FLOW_ID = "testFlow";
-  private static final String MULE_CLIENT_ID = "MuleClient";
-
   private static final int TIMEOUT = 1000;
-  private static final HttpRequestOptions GET_OPTIONS = HttpRequestOptionsBuilder.newOptions()
-      .method(HttpConstants.Methods.GET.name()).responseTimeout(TIMEOUT).disableStatusCodeValidation().build();
 
   @Rule
   public DynamicPort port = new DynamicPort("port");
 
+  @Rule
+  public TestHttpClient httpClient = new TestHttpClient.Builder(getService(HttpService.class)).build();
+
   @Override
   protected String getConfigFile() {
     return "org/mule/test/integration/notifications/connector-message-error-notification-test-flow.xml";
   }
 
   @Test
   public void doTest() throws Exception {
-    final String url = String.format("http://localhost:%s/path", port.getNumber());
-    muleContext.getClient().send(url, InternalMessage.of(TEST_PAYLOAD), GET_OPTIONS);
+    HttpRequest request =
+        HttpRequest.builder().setUri(String.format("http://localhost:%s/path", port.getNumber())).setMethod(POST.name()).build();
+
+    httpClient.send(request, TIMEOUT, false, null);
 
     assertNotifications();
   }
 
   @Override
   public RestrictedNode getSpecification() {
-    return new Node().parallel(new Node(ConnectorMessageNotification.class, MESSAGE_REQUEST_BEGIN, MULE_CLIENT_ID))
-        .parallel(new Node(ConnectorMessageNotification.class, MESSAGE_REQUEST_END, MULE_CLIENT_ID))
+    return new Node()
```

Comment:
Notification were genereated for mule client and flow. I removed the client ones